### PR TITLE
Main.py + Requirements.txt

### PR DESCRIPTION
**Pull request by jp-krow**
🎨 Version 1 of TL;DR

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    try:
+      LANGUAGE = request.form.get('site-language')
+
+      #sentences parameter optional, defaults to 10
+      if request.form.get("sentences",None) == None:
+        SENTENCES_COUNT = 10
+      else:
+        SENTENCES_COUNT = int(request.form.get('sentences'))
+
+      #optional google cashed parameter, helps avoid anti-bot pages
```
Comment (EnderDev) [inline main.py:42]: misspelling of cached

**Review by EnderDev — CHANGES_REQUESTED** [review]
Looks good, just fix the spelling mistake and I can merge 👍

**Comment by jp-krow** [comment]
Fixed typo, also added README.

**Comment by jp-krow** [comment]
🎨 idk how to use these emojis

File: README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+`cached` (Optional) enables or disables google's cached save of the url (useful if the website has anti-bot measures in place, or keep receiving an error.)
+
+## Contributing
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
+
+Please make sure to update tests as appropriate.
+
+## License
+[MIT](https://choosealicense.com/licenses/mit/)
```
Comment (EnderDev) [inline README.md:45]: We aren't using MIT, we are using MPL-2.0

**Comment by xMinota** [comment]
tl;dr?

**Pull request merged**